db/migrate/20230117150122_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.string :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
your table should contain references `this will create author_id and a corresponding index (but you can also add column and index in separate commands)` 😸